examples/demo-stf/src/runtime.rs

```diff
@@ -1,11 +1,11 @@
 #[cfg(feature = "native")]
-use sov_accounts::{AccountsRpcImpl, AccountsRpcServer};
+use sov_accounts::query::{AccountsRpcImpl, AccountsRpcServer};
```

Comment:
Synced with @bkolad  offline. Summarizing here: 

The motivation for this change is that we were forgetting to export a bunch of the generated types that users need when they want to actually query the module.  (i.e. _RpcClient, AccountsResponse). Since some of those types are macro generated, I thought it would be cleaner to export the query module instead of expecting module developers export those types manually. 

After talking it through, we think it will be even cleaner to  have the rpc macro generate a new mod  named something like generated_public_types and add a pub use query::generated_public_types::*  instead.

Reply:
Opened #713